codice/ddf
# DDF-3628 - Updates Intrigue searches to be done through websockets

#### What does this PR do?

Updates Intrigue searches to be done through websockets.

#### Who is reviewing it? 

@adimka 
@mackncheesiest 

#### Select relevant component teams: 

@codice/ui 
@codice/security 

#### Choose 2 committers to review/merge the PR.

@andrewkfiedler
@pklinef

#### How should this be tested?

- Full build of catalog-ui-search
- Tries queries with no/some data ingested in the system
- Ensure development through the proxy still works

#### What are the relevant tickets?

[DDF-3628](https://codice.atlassian.net/browse/DDF-3628)

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
